.github/workflows/go.yml

```diff
@@ -0,0 +1,45 @@
+name: Go
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+env:
+  TESTFLAGS: -v
+  BUILDFLAGS: -v
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        go-version: [1.20.x, 1.21.x]
+        os: [ubuntu-latest]
```

Comment:
This dropped the other platforms (windows, mac)

Reply:
Do we need them? They seemed like overkill.

Reply:
The package is used on platforms other than linux, so, yes, at least I'd like to make sure it works.